SA/project_documentation/content/time_tracking.typ

```diff
@@ -0,0 +1,39 @@
+#import "@preview/cetz:0.1.2": canvas, chart
+
+= Time Tracking
+Content: expected effort & actual effort
+
+// (Label, Time Spent Lukas, Time Spent Jann)
```

Comment:
Unfortunately, `cetz` doesn't seem support legends. I feel like we will need that to specify which color is for whom.

We could also make the chart smaller vertically (works great, I tried it out) in order to fit two above each other. Another way to do it could be to do it side by side

Another approach altogether could be to re-use the charting stuff from the SE-Project, and just check it in through LFS. That would allow for some more charts, but also seems like more work.

What do you think?